authorizer/authorize_find.go

```diff
@@ -0,0 +1,245 @@
+package authorizer
```

Comment:
@affo I forgot I looked at this yesterday and it looks good, but my only question was if we could remove some of the duplication in this file... unfortunately because the loops are over slices of different types, it doesn't look like it would be easy 😞

Reply:
Yes, that would be great!

I did generate this using a template and fixing particular cases by hand.

A good thing would be  to have and `IDer` interface that exposes `ID()` and `OrgID()` and make every resource implement that. At that point generation would only be a metter of iterating over resources that implement `IDer` and generate this file.

I can think about do it in the future 👍